### ISSUE TITLE
Change Emoji position

### ISSUE THREAD

**Issue by LauraEhlis** [IRT-Open-Source/imsced#104]
If the user selected a subtitle block a small emoji appears on the right side of the subtitle line. 

![emoji 1](https://user-images.githubusercontent.com/59090140/85122342-b0076f00-b226-11ea-8d00-4f542016387d.jpg)

While clicking on the emoji the user opens a new window where a emoji can be selected for the corresponding subtitle.

![emoji 2](https://user-images.githubusercontent.com/59090140/85122359-b564b980-b226-11ea-84ff-33ed95b64ad4.jpg)

**Comment by tairt** [comment]
@LauraEhlis Yes, I like to bring this closer to the editing action. We may want to make it configurable if this is displayed or not. Emojis may not be used in every scenario/by every user.

**Comment by tairt** [comment]
The following recording shows the current problem this issue tries to resolve. 
![emoji-verbessern-imsced](https://user-images.githubusercontent.com/4597762/91074536-00db7f80-e63d-11ea-853b-6d2675bf2da5.gif)
